app/bundles/PluginBundle/Views/FormTheme/Integration/_integration_details_featureSettings_row.html.twig

```diff
@@ -1,15 +0,0 @@
-{% block _integration_details_featureSettings_row %}
-<div class="row">
-    <div class="col-sm-12">
-        <h4 class="mb-sm mt-lg">
-            {{ form.vars['label']|trans }}
-        </h4>
-        {% if formNotes.features is defined and formNotes.features is not empty %}
-            <div class="alert alert-{{ formNotes['features']['type'] }}">
-                {{ formNotes['features']['note']|trans }}
-            </div>
-        {% endif %}
-        {{ form_widget(form) }}
-    </div>
-</div>
-{% endblock %}
```

Comment:
Why is this twig file (and others like it) deleted?

Reply:
I found it easier to debug/troubleshoot issues when there is only one file vs many. This is also the same logic as https://github.com/mautic/mautic/blob/5.x/app/bundles/CoreBundle/Resources/views/FormTheme/mautic_form_layout.html.twig where multiple files were rolled up into one.